.github/ISSUE_TEMPLATE/2_improvement.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Improvement proposal
+about: Improvements to the contracts, the website, the copy... Anything that is working well but could be enhanced.
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+<!-- Thank you for using White Whale Migaloo!
+
+     If you are looking for support, please check out our documentation
+     or consider asking a question on Discord's smart contract channel:
+      * https://whitewhale.money/
+      * https://white-whale-defi-platform.github.io/docs/
+      * https://discordapp.com/channels/908044702794801233/987301947440767006
+
+     If you have found a bug or if our documentation doesn't have an answer
+     to what you're looking for, then fill out the template below.
+-->
+
+If you want to report a security issue, please follow our security policy: `https://github.com/White-Whale-Defi-Platform/migaloo-core/blob/main/SECURITY.md`
```

Comment:
Should this be included in the comments above?

Reply:
I think it's better to have it there so they don't miss it, in the comment they maybe miss it by mistake and report a security issue publicly

Reply:
Sounds reasonable